src/inc/msquicp.h

```diff
@@ -93,29 +93,8 @@ typedef struct QUIC_PRIVATE_TRANSPORT_PARAMETER {
     const uint8_t* Buffer;
 } QUIC_PRIVATE_TRANSPORT_PARAMETER;
 
-//
-// This struct enables QUIC applications to support SSLKEYLOGFILE
-// for debugging packet captures with e.g. Wireshark.
-//
-
-#define CXPLAT_TLS_SECRETS_MAX_SECRET_LEN 64
-typedef struct CXPLAT_TLS_SECRETS {
-    uint8_t SecretLength;
-    struct {
-        uint8_t ClientRandom : 1;
-        uint8_t ClientEarlyTrafficSecret : 1;
-        uint8_t ClientHandshakeTrafficSecret : 1;
-        uint8_t ServerHandshakeTrafficSecret : 1;
-        uint8_t ClientTrafficSecret0 : 1;
-        uint8_t ServerTrafficSecret0 : 1;
-    } IsSet;
-    uint8_t ClientRandom[32];
-    uint8_t ClientEarlyTrafficSecret[CXPLAT_TLS_SECRETS_MAX_SECRET_LEN];
-    uint8_t ClientHandshakeTrafficSecret[CXPLAT_TLS_SECRETS_MAX_SECRET_LEN];
-    uint8_t ServerHandshakeTrafficSecret[CXPLAT_TLS_SECRETS_MAX_SECRET_LEN];
-    uint8_t ClientTrafficSecret0[CXPLAT_TLS_SECRETS_MAX_SECRET_LEN];
-    uint8_t ServerTrafficSecret0[CXPLAT_TLS_SECRETS_MAX_SECRET_LEN];
-} CXPLAT_TLS_SECRETS;
+#define CXPLAT_TLS_SECRETS_MAX_SECRET_LEN QUIC_TLS_SECRETS_MAX_SECRET_LEN
```

Comment:
Do we still need these? We're already binary breaking consumers by changing the param id, and this file isn't a public api file anyway.

Reply:
We don't strictly need them. This was just to make sure anything that was building against these continued to compile. I'm fine removing them if you think we should.

Reply:
I don't like doing a binary breaking change without also doing a source breaking change. Makes it much easier from a debugging standpoint to figure out why something is broken, as a program rebuild will tell you something changed. If you want to keep this, then we need the existing param id to still work at a binary level.

Reply:
Removed.